tests/ImageSharp.Web.Tests/Helpers/ExifOrientationUtilitiesTests.cs

```diff
@@ -0,0 +1,139 @@
+                { AnchorPositionMode.Right, ExifOrientationMode.RightTop, AnchorPositionMode.Top },
+                { AnchorPositionMode.Right, ExifOrientationMode.RightBottom, AnchorPositionMode.Top },
+                { AnchorPositionMode.TopLeft, ExifOrientationMode.TopRight, AnchorPositionMode.TopRight },
+                { AnchorPositionMode.TopLeft, ExifOrientationMode.LeftBottom, AnchorPositionMode.TopRight },
+                { AnchorPositionMode.TopLeft, ExifOrientationMode.BottomRight, AnchorPositionMode.BottomLeft },
+                { AnchorPositionMode.TopLeft, ExifOrientationMode.RightTop, AnchorPositionMode.BottomLeft },
+                { AnchorPositionMode.TopLeft, ExifOrientationMode.BottomLeft, AnchorPositionMode.BottomRight },
+                { AnchorPositionMode.TopLeft, ExifOrientationMode.RightBottom, AnchorPositionMode.BottomRight },
+                { AnchorPositionMode.TopRight, ExifOrientationMode.TopRight, AnchorPositionMode.TopLeft },
+                { AnchorPositionMode.TopRight, ExifOrientationMode.RightTop, AnchorPositionMode.TopLeft },
+                { AnchorPositionMode.TopRight, ExifOrientationMode.BottomLeft, AnchorPositionMode.BottomRight },
+                { AnchorPositionMode.TopRight, ExifOrientationMode.LeftBottom, AnchorPositionMode.BottomRight },
+                { AnchorPositionMode.TopRight, ExifOrientationMode.BottomRight, AnchorPositionMode.BottomLeft },
+                { AnchorPositionMode.TopRight, ExifOrientationMode.LeftTop, AnchorPositionMode.BottomLeft },
+                { AnchorPositionMode.BottomRight, ExifOrientationMode.TopRight, AnchorPositionMode.BottomLeft },
+                { AnchorPositionMode.BottomRight, ExifOrientationMode.LeftBottom, AnchorPositionMode.BottomLeft },
+                { AnchorPositionMode.BottomRight, ExifOrientationMode.BottomLeft, AnchorPositionMode.TopRight },
+                { AnchorPositionMode.BottomRight, ExifOrientationMode.RightTop, AnchorPositionMode.TopRight },
+                { AnchorPositionMode.BottomRight, ExifOrientationMode.BottomRight, AnchorPositionMode.TopLeft },
+                { AnchorPositionMode.BottomRight, ExifOrientationMode.RightBottom, AnchorPositionMode.TopLeft },
+                { AnchorPositionMode.BottomLeft, ExifOrientationMode.TopRight, AnchorPositionMode.BottomRight },
+                { AnchorPositionMode.BottomLeft, ExifOrientationMode.RightTop, AnchorPositionMode.BottomRight },
+                { AnchorPositionMode.BottomLeft, ExifOrientationMode.BottomLeft, AnchorPositionMode.TopLeft },
+                { AnchorPositionMode.BottomLeft, ExifOrientationMode.LeftBottom, AnchorPositionMode.TopLeft },
+                { AnchorPositionMode.BottomLeft, ExifOrientationMode.BottomRight, AnchorPositionMode.TopRight },
+                { AnchorPositionMode.BottomLeft, ExifOrientationMode.LeftTop, AnchorPositionMode.TopRight },
```

Comment:
And this the complete matrix of `AnchorPositionMode` and `ExifOrientationMode` to test:
```suggestion
                { AnchorPositionMode.Center, ExifOrientationMode.Unknown, AnchorPositionMode.Center },
                { AnchorPositionMode.Center, ExifOrientationMode.TopLeft, AnchorPositionMode.Center },
                { AnchorPositionMode.Center, ExifOrientationMode.TopRight, AnchorPositionMode.Center },
                { AnchorPositionMode.Center, ExifOrientationMode.BottomRight, AnchorPositionMode.Center },
                { AnchorPositionMode.Center, ExifOrientationMode.BottomLeft, AnchorPositionMode.Center },
                { AnchorPositionMode.Center, ExifOrientationMode.LeftTop, AnchorPositionMode.Center },
                { AnchorPositionMode.Center, ExifOrientationMode.RightTop, AnchorPositionMode.Center },
                { AnchorPositionMode.Center, ExifOrientationMode.RightBottom, AnchorPositionMode.Center },
                { AnchorPositionMode.Center, ExifOrientationMode.LeftBottom, AnchorPositionMode.Center },
                { AnchorPositionMode.Top, ExifOrientationMode.Unknown, AnchorPositionMode.Top },
                { AnchorPositionMode.Top, ExifOrientationMode.TopLeft, AnchorPositionMode.Top },
                { AnchorPositionMode.Top, ExifOrientationMode.TopRight, AnchorPositionMode.Top },
                { AnchorPositionMode.Top, ExifOrientationMode.BottomRight, AnchorPositionMode.Bottom },
                { AnchorPositionMode.Top, ExifOrientationMode.BottomLeft, AnchorPositionMode.Bottom },
                { AnchorPositionMode.Top, ExifOrientationMode.LeftTop, AnchorPositionMode.Left },
                { AnchorPositionMode.Top, ExifOrientationMode.RightTop, AnchorPositionMode.Left },
                { AnchorPositionMode.Top, ExifOrientationMode.RightBottom, AnchorPositionMode.Right },
                { AnchorPositionMode.Top, ExifOrientationMode.LeftBottom, AnchorPositionMode.Right },
                { AnchorPositionMode.Bottom, ExifOrientationMode.Unknown, AnchorPositionMode.Bottom },
                { AnchorPositionMode.Bottom, ExifOrientationMode.TopLeft, AnchorPositionMode.Bottom },
                { AnchorPositionMode.Bottom, ExifOrientationMode.TopRight, AnchorPositionMode.Bottom },
                { AnchorPositionMode.Bottom, ExifOrientationMode.BottomRight, AnchorPositionMode.Top },
                { AnchorPositionMode.Bottom, ExifOrientationMode.BottomLeft, AnchorPositionMode.Top },
                { AnchorPositionMode.Bottom, ExifOrientationMode.LeftTop, AnchorPositionMode.Right },
                { AnchorPositionMode.Bottom, ExifOrientationMode.RightTop, AnchorPositionMode.Right },
                { AnchorPositionMode.Bottom, ExifOrientationMode.RightBottom, AnchorPositionMode.Left },
                { AnchorPositionMode.Bottom, ExifOrientationMode.LeftBottom, AnchorPositionMode.Left },
                { AnchorPositionMode.Left, ExifOrientationMode.Unknown, AnchorPositionMode.Left },
                { AnchorPositionMode.Left, ExifOrientationMode.TopLeft, AnchorPositionMode.Left },
                { AnchorPositionMode.Left, ExifOrientationMode.TopRight, AnchorPositionMode.Right },
                { AnchorPositionMode.Left, ExifOrientationMode.BottomRight, AnchorPositionMode.Right },
                { AnchorPositionMode.Left, ExifOrientationMode.BottomLeft, AnchorPositionMode.Left },
                { AnchorPositionMode.Left, ExifOrientationMode.LeftTop, AnchorPositionMode.Top },
                { AnchorPositionMode.Left, ExifOrientationMode.RightTop, AnchorPositionMode.Bottom },
                { AnchorPositionMode.Left, ExifOrientationMode.RightBottom, AnchorPositionMode.Bottom },
                { AnchorPositionMode.Left, ExifOrientationMode.LeftBottom, AnchorPositionMode.Top },
                { AnchorPositionMode.Right, ExifOrientationMode.Unknown, AnchorPositionMode.Right },
                { AnchorPositionMode.Right, ExifOrientationMode.TopLeft, AnchorPositionMode.Right },
                { AnchorPositionMode.Right, ExifOrientationMode.TopRight, AnchorPositionMode.Left },
                { AnchorPositionMode.Right, ExifOrientationMode.BottomRight, AnchorPositionMode.Left },
                { AnchorPositionMode.Right, ExifOrientationMode.BottomLeft, AnchorPositionMode.Right },
                { AnchorPositionMode.Right, ExifOrientationMode.LeftTop, AnchorPositionMode.Bottom },
                { AnchorPositionMode.Right, ExifOrientationMode.RightTop, AnchorPositionMode.Top },
                { AnchorPositionMode.Right, ExifOrientationMode.RightBottom, AnchorPositionMode.Top },
                { AnchorPositionMode.Right, ExifOrientationMode.LeftBottom, AnchorPositionMode.Bottom },
                { AnchorPositionMode.TopLeft, ExifOrientationMode.Unknown, AnchorPositionMode.TopLeft },
                { AnchorPositionMode.TopLeft, ExifOrientationMode.TopLeft, AnchorPositionMode.TopLeft },
                { AnchorPositionMode.TopLeft, ExifOrientationMode.TopRight, AnchorPositionMode.TopRight },
                { AnchorPositionMode.TopLeft, ExifOrientationMode.BottomRight, AnchorPositionMode.BottomRight },
                { AnchorPositionMode.TopLeft, ExifOrientationMode.BottomLeft, AnchorPositionMode.BottomLeft },
                { AnchorPositionMode.TopLeft, ExifOrientationMode.LeftTop, AnchorPositionMode.TopLeft },
                { AnchorPositionMode.TopLeft, ExifOrientationMode.RightTop, AnchorPositionMode.BottomLeft },
                { AnchorPositionMode.TopLeft, ExifOrientationMode.RightBottom, AnchorPositionMode.BottomRight },
                { AnchorPositionMode.TopLeft, ExifOrientationMode.LeftBottom, AnchorPositionMode.TopRight },
                { AnchorPositionMode.TopRight, ExifOrientationMode.Unknown, AnchorPositionMode.TopRight },
                { AnchorPositionMode.TopRight, ExifOrientationMode.TopLeft, AnchorPositionMode.TopRight },
                { AnchorPositionMode.TopRight, ExifOrientationMode.TopRight, AnchorPositionMode.TopLeft },
                { AnchorPositionMode.TopRight, ExifOrientationMode.BottomRight, AnchorPositionMode.BottomLeft },
                { AnchorPositionMode.TopRight, ExifOrientationMode.BottomLeft, AnchorPositionMode.BottomRight },
                { AnchorPositionMode.TopRight, ExifOrientationMode.LeftTop, AnchorPositionMode.BottomLeft },
                { AnchorPositionMode.TopRight, ExifOrientationMode.RightTop, AnchorPositionMode.TopLeft },
                { AnchorPositionMode.TopRight, ExifOrientationMode.RightBottom, AnchorPositionMode.TopRight },
                { AnchorPositionMode.TopRight, ExifOrientationMode.LeftBottom, AnchorPositionMode.BottomRight },
                { AnchorPositionMode.BottomRight, ExifOrientationMode.Unknown, AnchorPositionMode.BottomRight },
                { AnchorPositionMode.BottomRight, ExifOrientationMode.TopLeft, AnchorPositionMode.BottomRight },
                { AnchorPositionMode.BottomRight, ExifOrientationMode.TopRight, AnchorPositionMode.BottomLeft },
                { AnchorPositionMode.BottomRight, ExifOrientationMode.BottomRight, AnchorPositionMode.TopLeft },
                { AnchorPositionMode.BottomRight, ExifOrientationMode.BottomLeft, AnchorPositionMode.TopRight },
                { AnchorPositionMode.BottomRight, ExifOrientationMode.LeftTop, AnchorPositionMode.BottomRight },
                { AnchorPositionMode.BottomRight, ExifOrientationMode.RightTop, AnchorPositionMode.TopRight },
                { AnchorPositionMode.BottomRight, ExifOrientationMode.RightBottom, AnchorPositionMode.TopLeft },
                { AnchorPositionMode.BottomRight, ExifOrientationMode.LeftBottom, AnchorPositionMode.BottomLeft },
                { AnchorPositionMode.BottomLeft, ExifOrientationMode.Unknown, AnchorPositionMode.BottomLeft },
                { AnchorPositionMode.BottomLeft, ExifOrientationMode.TopLeft, AnchorPositionMode.BottomLeft },
                { AnchorPositionMode.BottomLeft, ExifOrientationMode.TopRight, AnchorPositionMode.BottomRight },
                { AnchorPositionMode.BottomLeft, ExifOrientationMode.BottomRight, AnchorPositionMode.TopRight },
                { AnchorPositionMode.BottomLeft, ExifOrientationMode.BottomLeft, AnchorPositionMode.TopLeft },
                { AnchorPositionMode.BottomLeft, ExifOrientationMode.LeftTop, AnchorPositionMode.TopRight },
                { AnchorPositionMode.BottomLeft, ExifOrientationMode.RightTop, AnchorPositionMode.BottomRight },
                { AnchorPositionMode.BottomLeft, ExifOrientationMode.RightBottom, AnchorPositionMode.BottomLeft },
                { AnchorPositionMode.BottomLeft, ExifOrientationMode.LeftBottom, AnchorPositionMode.TopLeft },
```

Reply:
I applied this test matrix to the current transform and got 2 failing tests:
```
CanTransformAnchor(anchor: TopLeft, orientation: 3, expected: BottomRight)
Assert.Equal() Failure
Expected: BottomRight
Actual:   BottomLeft

CanTransformAnchor(anchor: TopLeft, orientation: 4, expected: BottomLeft)
Assert.Equal() Failure
Expected: BottomLeft
Actual:   BottomRight
```

I believe the test cases are correct, so that would mean the current implementation contains a bug:
- Orientation 3 (bottom/right - upside down) should invert both horizontally and vertically, so top/left should become bottom/right
- Orientation 4 (bottom/left - flipped back-to-front and upside down) should only invert vertically, so top/left should become bottom/left

Reply:
There's a good chance you found a bug - orientation is a bit of a brain twister! I will review your matrix and implementation asap. Really appreciate the effort you're making here btw! 👍